src/game/Protocol/WorldSocket.cpp

```diff
@@ -3,6 +3,7 @@
  * Copyright (C) 2009-2011 MaNGOSZero <https://github.com/mangos/zero>
  * Copyright (C) 2011-2016 Nostalrius <https://nostalrius.org>
  * Copyright (C) 2016-2017 Elysium Project <https://github.com/elysium-project>
+ * Copyright (C) 2017-2024 VMaNGOS Project <https://github.com/vmangos>
```

Comment:
Since you've rewritten this file just remove the part with the old project names.

Reply:
It was heavily inspired by cMangos. <https://github.com/cmangos/mangos-classic/blob/master/src/game/Server/WorldSocket.cpp>, soooo 🤷‍♂️